Yona/Yona/Others/SMSValidPasscodeLogin/Passcode/ConfirmPasscodeViewController.swift

```diff
@@ -12,15 +12,37 @@ final class ConfirmPasscodeViewController:  LoginSignupValidationMasterView {
 
     var passcode: String?
 
+    @IBOutlet var screenNameLabel: UILabel!
+    @IBOutlet var topView: UIView!
+    @IBOutlet var avtarImage: UIImageView!
+    @IBOutlet var gradientContainerView: UIView!
+    var isFromSettings = false
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         //Nav bar Back button.
         self.navigationItem.hidesBackButton = true
         self.navigationController?.setNavigationBarHidden(true, animated: false)
-        
-        self.gradientView.colors = [UIColor.yiGrapeTwoColor(), UIColor.yiGrapeTwoColor()]
-        
+                
         UIApplication.sharedApplication().setStatusBarStyle(UIStatusBarStyle.LightContent, animated: false)
+        
+        if isFromSettings {
+            screenNameLabel.text = NSLocalizedString("change-pin", comment: "")
+            topView.backgroundColor = UIColor.yiMangoColor()
+            self.gradientView.colors = [UIColor.yiMangoTriangleColor(), UIColor.yiMangoTriangleColor()]
+            gradientContainerView.backgroundColor = UIColor.yiMangoColor()
+            
+            let viewWidth = self.view.frame.size.width
+            let customView=UIView(frame: CGRectMake(0, 0, (viewWidth-60)/3, 2))
+            customView.backgroundColor=UIColor.yiDarkishPinkColor()
+            self.progressView.addSubview(customView)
```

Comment:
See the above comment on the same code snippet in the LoginViewController.swift file